backend/src/main/java/ai/verta/modeldb/entities/ProjectEntity.java

```diff
@@ -120,6 +120,9 @@ public ProjectEntity(Project project) throws InvalidProtocolBufferException {
 
   @Transient private Map<String, List<ArtifactEntity>> artifactEntityMap = new HashMap<>();
 
+  @Column(name = "workspace_id")
```

Comment:
This will be so confusing =)
Maybe we change the internal names of "workspace" later. Not needed, just something to help my brain.

Reply:
It's verbose, but we could use `workspace_entity_id` to make it clear what this is.

Reply:
We can also rename `workspace` to `workspace_name` to clarify what it is.

Reply:
I don't think it's the name, but the legacy id (id for user or id for organization based on the type). We can keep the name as is, just something to be aware of.